content/engineering/enablement/delivery/managed/operations.md

```diff
@@ -131,6 +132,14 @@ default-nat-manual-ip-1  $NAT_IP_TWO     EXTERNAL                    us-central1
 - `$GLOBAL_IP` is the IP address that `$CUSTOMER.sourcegraph.com` should point to, it is the address of the GCP Load Balancer.
 - `$NAT_IP_ONE` and `$NAT_IP_TWO` are the external IPs from which egress traffic from the deployment will originate from. These are the addresses from which Sourcegraph will access the customer's code host, and as such the customer will need to allow them access to e.g. their internal code host.
 
+### Accessing the web-app of the instance
+
+This isn't highly technical, and just boils down to discovery.
+
+You can find the instance's web URL by going to [this GitHub repository](https://github.com/sourcegraph/deploy-sourcegraph-managed), and looking at the customer's `terraform.tfvars` file. The value listed under `domain` is the URL you can use to navigate to the web-app. As for actually logging in, you will need to have account credentials shared with you via 1password. Ask #it-tech-ops in Slack.
+
+If you are running into a 403 forbidden error, you may be running into an exception... See if [this](https://docs.google.com/document/d/1E0E2t2voiPHh4EioBBJpyB-nXbJihm3bCyIct-pY2mc/edit) document address your needs. If it doesn't, you may need to seek additional help.
```

Comment:
I would actually suggest we do the inverse. Always use `gcloud` to access customer instances. That should be our standard approach and is documented in other places in this document.